tests/unit/App.spec.js

```diff
@@ -0,0 +1,43 @@
+import { createLocalVue, shallowMount } from "@vue/test-utils";
+import { render } from "@testing-library/vue";
+
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+
+import App from "@/App";
+import HelloWorld from "@/components/HelloWorld.vue";
+
+// from https://github.com/testing-library/vue-testing-library/blob/master/src/__tests__/vuetify.js
+
+// Custom container to integrate Vuetify with Vue Testing Library.
+// Vuetify requires you to wrap your app with a v-app component that provides
+// a <div data-app="true"> node.
+const renderWithVuetify = (component, options, callback) => {
```

Comment:
Will probably want to make a util method?

Reply:
Is this really necessary? https://iobio.io/user_guides/2019/05/20/vue-js-unit-testing-with-jest/